src/proto/FunctionRpc.proto

```diff
@@ -553,6 +553,13 @@ message RpcException {
 
   // Textual message describing the exception
   string message = 2;
+
+  // Type of exception if thrown by user's function code 
+  optional string remote_type = 4; 
+
+  // Worker specifies whether exception is a user exception, 
+  // for purpose of application insights logging. 
+  optional bool is_user_exception = 5;
```

Comment:
Would this default to true or false if not specified? It would be helpful if you included that info in the comment in the code

Reply:
It would default to false - I will add that to the comment.